src/sentry/api/serializers/models/dashboard.py

```diff
@@ -52,12 +55,30 @@ def serialize(self, obj, attrs, user, **kwargs):
 
 
 class DashboardListSerializer(Serializer):
+    def get_attrs(self, item_list, user):
+        item_dict = {i.id: i for i in item_list}
+
+        widgets = list(
```

Comment:
Adding this because we need to access widget type when rendering the mini dashboards, yet want to keep this as lightweight as possible and not have to serialize the entire widget object